kernel/src/actions/mod.rs

```diff
@@ -454,12 +461,12 @@ mod tests {
         let expected = Arc::new(StructType::new(vec![StructField::new(
             "commitInfo",
             StructType::new(vec![
-                StructField::new("timestamp", DataType::LONG, false),
-                StructField::new("operation", DataType::STRING, false),
+                StructField::new("timestamp", DataType::LONG, true),
```

Comment:
So to my questions above, are you using these options as nullable fields than truly optional ones?

Reply:
`Option` in schema-land is just a nullable field so I think the answer is yes?